themes/theme-gmd/components/View/components/Content/components/Container/index.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
```

Comment:
I don't think that we need this new component. Instead of passing the ref to this component, the consumption of the ref could happen inside the <View> and then be passed down to the content component.

Reply:
done